src/utils/truth-counter.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file truth-counter.js
+ * @author Ubspy, josiezxh
+ * @desc
+  Whenever '+1' is detected as a message, it will check if it is a reply, and add to the truth counter channel.
+  If it isn't, then won't add to counter. If the '+1' doesn't work, check the
+  logger.
+ *
+ * @param {discord.js <Message>}, Message [Message object to work with]
+ {@link} https://discord.js.org/#/docs/main/stable/class/Message
+*/
+
+const fs = require('fs');
+const config = require('../../config/config.json'); // Goes 2 folders back to get config file.
+const truthCounter = require('../../config/truthCounter.json');
+
+// Returns a function with parameters client and logger.
+module.exports = (client, logger) => {
+
```

Comment:
there's a lot of unnecessary spacing in this file :(
it would be best to get rid of the new lines where they aren't needed